polar_route/cli.py

```diff
@@ -2,16 +2,19 @@
 import json
 import inspect
 import logging
+import fiona
```

Comment:
I am a bit worried if Fiona is required as an independent package. It's included if GeoPandas is loaded, but this could cause issues on HPC/Pipeline stuff with GDAL. 

I would be good to only have these within the path_kml function so that they are only called on a specific use case where path in KML format is required.

@hjabbot what do you think ?

Reply:
Yeah I don't really mind although I don't think testing so far has thrown up any issues with this. The only time I had any errors was when Fiona was missing from the requirements and I ran on an install where geopandas was using a different backend so Fiona wasn't available. The main issue is if you want to save KML files from geopandas you seem to need to explicitly enable the matching driver. If anyone wants to look for a better solution I'm open to ideas, as I said on the [original PR](https://github.com/antarctica/PolarRoute/pull/287#issuecomment-2291192240).

Importing within a function is technically not recommended by PEP8 but it's fine to break with convention in a few places. Removing it from the requirements is probably more likely to lead to issues though because then Fiona might just not be installed depending on the platform PolarRoute is installed on.